src/gwt/acesupport/acemode/yaml_highlight_rules.js

```diff
@@ -90,16 +94,14 @@ define("mode/yaml_highlight_rules", ["require", "exports", "module"], function (
             regex: "[&\\*][a-zA-Z0-9-_]+"
          },
          {
+            // (package) (::) (function) (:)
             token: ["meta.tag", "keyword", "meta.tag", "keyword.operator"],
-            regex: /^(\s*[\w\-].*?)(:{2,3})(\s*[\w\-].*?)(:(?:\s+|$))/
```

Comment:
I'm fairly certain the use of `.*` here was a typo; I can't imagine why we'd want to a catch-all regular expression here? That seems too permissive?